packages/flutter/lib/src/services/restoration.dart

```diff
@@ -312,6 +312,7 @@ class RestorationManager extends ChangeNotifier {
       default:
         throw UnimplementedError("${call.method} was invoked but isn't implemented by $runtimeType");
     }
+    return null;
```

Comment:
Isn't this dead code? The switch above should handle all possible cases given the `default` case. So I would expect you wouldn't need `return null` here.

Reply:
`break` will jump out of the switch, hitting the implicit return.

Reply:
Ah! I missed the break. I think this method should just be typed with a return type of `Future<void>`. That would remove the need for the return, presumably?

Reply:
done